Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -535,6 +556,32 @@ void RomBrowserWidget::on_listViewWidget_sectionMoved(int logicalIndex, int oldV
     CoreSettingsSetValue(SettingsID::RomBrowser_ColumnOrder, columnOrder);
 }
 
+void RomBrowserWidget::on_listViewWidget_headerContextMenuRequested(QPoint position)
+{
+    this->headerContextMenu->clear();
+    RomBrowserWidget::on_listViewWidget_columnVisibilityMenuRequested(headerContextMenu);
+    this->headerContextMenu->popup(this->mapToGlobal(position));
+}
+
+void RomBrowserWidget::on_listViewWidget_columnVisibilityMenuRequested(QMenu* currentMenu)
```

Comment:
This is not a real signal slot function so it shouldn't be named as such, maybe a name like `addColumnsToMenu` would be better.

Reply:
Done, will include in next commit. (It's been renamed to "generateColumnsMenu").